ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_basic_operations.py

```diff
@@ -0,0 +1,40 @@
+# global
+import numpy as np
+from hypothesis import given
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# shape
+@handle_cmd_line_args
+@given(
+    xs_n_input_dtypes_n_unique_idx=helpers.dtype_and_values(available_dtypes=ivy_np.valid_dtypes),
+    as_variable=helpers.array_bools(),
```

Comment:
add `num_arrays=1` as a parameter to `array_bools` for `as_variable` and `native_shape`